drivers/ethernet/eth_native_posix.c

```diff
@@ -51,7 +51,7 @@ struct eth_context {
 	uint8_t mac_addr[6];
```

Comment:
Please change the commit subject to `drivers: eth: native_posix: Read back Ethernet interface name`
Also IMHO we should emphasize that it is the host kernel we are talking about here.

Reply:
Will do.